### PR TITLE
Oluwaseyi's card

### PR DESCRIPTION
**Pull request by emperor6inch**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -115,6 +115,38 @@ <h2>A project for first-time contributions</h2>
         <!-- make sure there is one line of space above and below your card -->
         <!-- ========== ⬇⬇ Paste YOUR CARD directly BELOW this line ⬇⬇ ========== -->
 
+                <!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________  -->
```
Comment (dilshankarunarathne) [inline index.html:118]: Please change this comment 
`<!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________ -->` 
into `<!-- ________ Your_Name Contributor card START ________ -->`

File: index.html [hunk 2]
```diff
@@ -115,6 +115,38 @@ <h2>A project for first-time contributions</h2>
...
+                      </li>
+                      <li>
+                        <a href="#" target="_blank" title="Third resource">
+                          Automating the Boring stuff with Python
+                        </a>
+                      </li>
+                    </ul>
+                  </div>
+                </div>
+                <!-- ________*END TEMPLATE* Contributor card END ________  -->
```
Comment (dilshankarunarathne) [inline index.html:147]: Please change this comment as well. 
`<!-- ________*END TEMPLATE* Contributor card END ________ -->` 
to `<!-- ________ Your_Name Contributor card END ________ -->`

**Review by dilshankarunarathne — CHANGES_REQUESTED** [review]
Hello @emperor6inch, thank you for contributing to this project. ❤
But there are a couple of small changes that need to be done before we go ahead and merge your code. Please change these two comments, because they are very helpful for everyone to identify each card. 
Once you've made those changes, please get back to us. We will be able to review and merge your pull request in a flash.

**Review by emperor6inch — COMMENTED** [review]
Thank you so much

**Review by dilshankarunarathne — APPROVED** [review]

**Comment by dilshankarunarathne** [comment]
Congratulations 🎉❤ @emperor6inch.
Your pull request was merged. 
You will be able to see your card on the [project website](https://syknapse.github.io/Contribute-To-This-Project/).

**Pull request merged**